packages/appChrome/components/SidebarItem.tsx

```diff
@@ -2,21 +2,53 @@ import * as React from "react";
 import { cx } from "emotion";
 import { appChromeInsetContent, sidebarNavItem } from "../style";
 import Clickable from "../../clickable/components/clickable";
-import { tintContentPrimary } from "../../shared/styles/styleUtils";
+import { tintContentPrimary, padding } from "../../shared/styles/styleUtils";
 
 export interface SidebarItemProps {
   children?: React.ReactElement<HTMLElement> | string;
   icon?: React.ReactElement<HTMLElement> | string;
   isActive?: boolean;
   onClick: (event?: React.SyntheticEvent<HTMLElement>) => void;
+  paddingBottom?:
```

Comment:
Left and right padding should also be customizable

Reply:
Okay, updated the jira to reflect this.